Tests/Indicators/PivotPointsHighLowTests.cs

```diff
@@ -117,6 +117,95 @@ public void PivotPointPerType(PivotPointType pointType)
             }
         }
 
+        [Test]
+        public void StrictCheckingDefaultBehavior()
+        {
+            // Test that default behavior (strictChecking = true) works as before
+            var pointsHighLow = new PivotPointsHighLow(2);
+
+            // Create data where middle point equals neighbors - should NOT be a pivot with strict checking
+            var bars = new[]
+            {
+                new TradeBar(DateTime.UtcNow.AddSeconds(0), Symbols.AAPL, 1, 10, 1, 1, 1), // high=10, low=1
+                new TradeBar(DateTime.UtcNow.AddSeconds(1), Symbols.AAPL, 1, 10, 1, 1, 1), // high=10, low=1 (same as middle)
+                new TradeBar(DateTime.UtcNow.AddSeconds(2), Symbols.AAPL, 1, 10, 1, 1, 1), // high=10, low=1 (middle - same as neighbors)
+                new TradeBar(DateTime.UtcNow.AddSeconds(3), Symbols.AAPL, 1, 10, 1, 1, 1), // high=10, low=1 (same as middle)
+                new TradeBar(DateTime.UtcNow.AddSeconds(4), Symbols.AAPL, 1, 10, 1, 1, 1), // high=10, low=1 (same as middle)
+            };
+
+            foreach (var bar in bars)
+            {
+                pointsHighLow.Update(bar);
+            }
+
+            // With strict checking (default), no pivot points should be detected since all values are equal
+            var pivotPoints = pointsHighLow.GetAllPivotPointsArray();
+            Assert.AreEqual(0, pivotPoints.Length, "With strict checking, equal values should not create pivot points");
+        }
+
+        [Test]
+        public void RelaxedCheckingAllowsTies()
+        {
+            // Test that relaxed checking (strictChecking = false) allows ties
+            var pointsHighLow = new PivotPointsHighLow(2, strictChecking: false);
+
+            // Create data where middle point equals neighbors - should be a pivot with relaxed checking
+            var bars = new[]
+            {
+                new TradeBar(DateTime.UtcNow.AddSeconds(0), Symbols.AAPL, 1, 10, 1, 1, 1), // high=10, low=1
+                new TradeBar(DateTime.UtcNow.AddSeconds(1), Symbols.AAPL, 1, 10, 1, 1, 1), // high=10, low=1 (same as middle)
+                new TradeBar(DateTime.UtcNow.AddSeconds(2), Symbols.AAPL, 1, 10, 1, 1, 1), // high=10, low=1 (middle - same as neighbors)
+                new TradeBar(DateTime.UtcNow.AddSeconds(3), Symbols.AAPL, 1, 10, 1, 1, 1), // high=10, low=1 (same as middle)
+                new TradeBar(DateTime.UtcNow.AddSeconds(4), Symbols.AAPL, 1, 10, 1, 1, 1), // high=10, low=1 (same as middle)
+            };
+
+            foreach (var bar in bars)
+            {
+                pointsHighLow.Update(bar);
+            }
+
+            // With relaxed checking, pivot points should be detected even with equal values
+            var pivotPoints = pointsHighLow.GetAllPivotPointsArray();
+            Assert.IsTrue(pivotPoints.Length > 0, "With relaxed checking, equal values should create pivot points");
```

Comment:
At least RelaxedCheckingAllowsTies and StrictCheckingDefaultBehavior should be test cases of the same test, see other `TestCase` usages,